src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/XamarinAndroidApplicationProject.cs

```diff
@@ -72,7 +72,7 @@ public XamarinAndroidApplicationProject (string debugConfigurationName = "Debug"
 			JavaPackageName = JavaPackageName ?? PackageName.ToLowerInvariant ();
 
 			OtherBuildItems.Add (new BuildItem.NoActionResource ("Properties\\AndroidManifest.xml") {
-				TextContent = ProcessManifestTemplate,
+				TextContent = () => { return ProcessManifestTemplate (); },
```

Comment:
Do we need this change?

Reply:
Yes. Without this change the `${PACKAGENAME}` value in the default manifest gets replaced with the default PackageName property. As a result the code I have to replace the package with the placeholder never works.

Reply:
I think the code is the same, the first one would be the same as doing:

```csharp
TextContent = new Func<string> (ProcessManifestTemplate)
```

I've heard putting `() => ProcessManifestTemplate()` prevents a delegate object from being created, but that performance shouldn't be needed here.

Reply:
The new test where I replace the code in the manifest with a Placeholder did not work with the old `TestContent` code. I'll take it out and see if I can figure out why the replacement was not happening.

Reply:
I backed out this change and it still worked... hey ho. Must have been a typo in my replacement code or something when I was testing.